server/server.js

```diff
@@ -492,47 +494,47 @@ async function checkHeartbeat() {
   try {
     const hubs = await db.getHubs()
     for (const hub of hubs) {
-      const currentTime = moment()
-      const flicLastSeenTime = moment(hub.flicLastSeenTime)
-      const flicDelayMillis = currentTime.diff(flicLastSeenTime)
-      const heartbeatLastSeenTime = moment(hub.heartbeatLastSeenTime)
-      const heartbeatDelayMillis = currentTime.diff(heartbeatLastSeenTime)
-      const pingLastSeenTime = moment(hub.flicLastPingTime)
-      const pingDelayMillis = currentTime.diff(pingLastSeenTime)
-
-      if (flicDelayMillis > FLIC_THRESHOLD_MILLIS && !hub.sentAlerts) {
+      const responderPhoneNumber = hub.installation.responderPhoneNumber
+      const currentTime = db.getCurrentTime()
+      const flicLastSeenTime = hub.flicLastSeenTime
+      const flicDelayMillis = currentTime - flicLastSeenTime
+      const heartbeatLastSeenTime = hub.heartbeatLastSeenTime
+      const heartbeatDelayMillis = currentTime - heartbeatLastSeenTime
+      const pingLastSeenTime = hub.flicLastPingTime
+      const pingDelayMillis = currentTime - pingLastSeenTime
+
+      const flicHeartbeatExceeded = flicDelayMillis > helpers.getEnvVar('HEARTBEAT_THRESHOLD_SECONDS')
+      const pingHeartbeatExceeded = pingDelayMillis > helpers.getEnvVar('HEARTBEAT_THRESHOLD_SECONDS')
+      const piHeartbeatExceeded = heartbeatDelayMillis > helpers.getEnvVar('HEARTBEAT_THRESHOLD_SECONDS')
```

Comment:
Different thresholds for us to get sentry alerts vs clients getting a text message